src/server/pps/server/worker_rc.go

```diff
@@ -251,6 +252,14 @@ func (a *apiServer) getWorkerOptions(pipelineName string, pipelineVersion uint64
 		Name:  client.PPSSpecCommitEnv,
 		Value: specCommitID,
 	})
+	workerEnv = append(workerEnv, v1.EnvVar{
+		Name: client.PProfPortEnv,
+		Value: fmt.Sprintf("%d", a.pprofPort),
+	})
+	workerEnv = append(workerEnv, v1.EnvVar{
+		Name: client.PeerPortEnv,
+		Value: fmt.Sprintf("%d", a.peerPort),
+	})
```

Comment:
Very minor nit, but it's probably better to use `strconv` for this, if for no other reason than to save an import.

Reply:
Yeah, I was avoiding that because `peerPort` is a uint16, which means I'd need to coerce the value, then call `FormatUint` or `Itoa`. I can do that though.